website/plugins.js

```diff
@@ -13,6 +14,7 @@ module.exports = [
     name: 'Analytics',
     pub: 'firebase_analytics',
     firebase: 'analytics',
+    documentation:'https://firebase.flutter.dev/docs/analytics/overview',
```

Comment:
We can probably make these relative links.

Reply:
Lets do this in another PR.